operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -171,12 +171,6 @@
   when:
     kiali_vars.external_services.grafana.in_cluster_url == ""
 
```

Comment:
I guess we should do the same inference than in Grafana for namespace in `in_cluster_url` , no? (basically copy/paste/modify the block above)